README.md

```diff
@@ -23,6 +23,8 @@ markdown so feel free to just list your project name or add a link, and make a
 pull request. You should even be able to [edit it
 online](https://github.com/python3statement/python3statement.github.io/edit/master/_sections/30-projects.md).
 
+There is no need to install jekyll, though: A check creating a preview will be run on your pull-request. Please check this preview.
```

Comment:
Nitpickery:

- jekyll -> Jekyll (based on https://jekyllrb.com/ ).
- I'd skip the word 'though' - it would make sense in the section above, but here I think it reads better without that.

Reply:
And pull-request -> pull request

Reply:
I just updated the text accordingly.